converter.go

```diff
@@ -1231,6 +1231,9 @@ func buildMapValues[K comparable, Vnullable any, VnotNullable any](mapValuesNull
 	}
 	result := make(map[K]VnotNullable, len(m))
 	for k, v := range m {
+		if v == nil {
```

Comment:
But nils are ok, as long as we talk about pointer types, right?

Reply:
Please have a look at how I handled this